Documentation/Environment.rst

```diff
@@ -8,3 +8,63 @@ Development environment
 
 You can run the code quality checks and automated tests locally (using a
 local PHP, Composer, and database) or using runTests.sh.
+
+To kickstart the project we suggest the usage of the `TYPO3-testing-distribution <https://github.com/oliverklee/TYPO3-testing-distribution/>`__ by Oliver Klee
```

Comment:
Please generally  let's keep the lines in the RST files to below 80 characters if possible.